### ISSUE TITLE
Remove spacing from Pan-India directory located in Maps.

### ISSUE THREAD

**Issue by rach-con13** [reach4help/reach4help#1350]
**Describe the bug**

When cloning the reach4help repo, an error  is caused by an invalid path in the pan-india directory. Which results in an empty cloned repo.

**To Reproduce**
1. Clone reach4help using the command : git clone 
2. Error message should pop up as seen below
```
`error: invalid path 'map/parse_scripts/india/pan_india/parsed_output_2021-05-26 13:02:04.638844/Delhi_Delhi_clean.csv'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.`
3. cd reach4help - directory should be empty.

**Comment by shayantist** [comment]
Good catch! The issue seems to be with [MS-DOS not allowing colons in file/folder names](https://www.reddit.com/r/explainlikeimfive/comments/4m1cem/eli5_why_cant_you_use_a_colon_in_the_name_while/) (*oh Windows*). Thanks @ethanstrominger for renaming the folder; I'll update the script that generates that folder so it doesn't happen again.

**Comment by ethanstrominger** [comment]
@schowdhury8 If these are generated, should they be added to .gitignore?  This could be a good "GOOD FIRST ISSUE" issue for a developer.  Also, the files that are inside the directory should not have spaces and colons.
